packages/components/src/tabs/styles.ts

```diff
@@ -104,65 +102,87 @@ export const TabListWrapper = styled.div`
 				${ COLORS.theme.accent };
 		}
 	}
-	&[aria-orientation='vertical']::before {
-		top: 0;
-		left: 0;
-		width: 100%;
-		height: calc( var( --antialiasing-factor ) * 1px );
-		transform: translateY( calc( var( --selected-top, 0 ) * 1px ) )
-			scaleY(
+	&[aria-orientation='vertical'] {
+		&::before {
+			/* Adjusting the border radius to match the scaling in the y axis. */
+			border-radius: ${ CONFIG.radiusSmall } /
 				calc(
-					var( --selected-height, 0 ) / var( --antialiasing-factor )
-				)
+					${ CONFIG.radiusSmall } /
+						(
+							var( --selected-height, 0 ) /
+								var( --antialiasing-factor )
+						)
+				);
+			top: 0;
+			left: 0;
+			width: 100%;
+			height: calc( var( --antialiasing-factor ) * 1px );
+			transform: translateY( calc( var( --selected-top, 0 ) * 1px ) )
+				scaleY(
+					calc(
+						var( --selected-height, 0 ) /
+							var( --antialiasing-factor )
+					)
+				);
+			background-color: color-mix(
+				in srgb,
+				${ COLORS.theme.accent },
+				transparent 96%
 			);
-		background-color: ${ COLORS.theme.gray[ 100 ] };
+		}
+		&[data-select-on-move='true']:has(
+				:is( :focus-visible, [data-focus-visible] )
```

Comment:
This is probably fixed by https://github.com/ariakit/ariakit/pull/4094 which is included in `@ariakit/react` `0.4.11`.

We should follow-up to update ariakit-related dependencies, and consequently we should be able to remove `:focus-visible` 🤞